src/scale.js

```diff
@@ -0,0 +1,73 @@
+var WebAnimationsPolyfillExtension;
+
+function isNumeric (number) {
+  return !isNaN(number);
+}
+
+function InvalidArgument (message) {
+  this.message = message;
+  this.name = 'InvalidArgument';
+}
+
+function parse (input) {
+  if (input === undefined || input === 'none') {
+    return null;
```

Comment:
You want to be able to animate none with a value e.g. none -> 8. Please add a test for that.